app/konnect/org-management/system-accounts.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Manage System Accounts
+content_type: how-to
+no_version: true
+---
+
+This guide explains what a system account is, how it varies from a user account, and how to manage a system account using the [{{site.konnect_short_name}} API](https://developer.konghq.com/spec/5175b87f-bfae-40f6-898d-82d224387f9b/fc735302-d8ac-4e66-a9ef-225569a75d3c). 
+
+A system account is a service account in {{site.konnect_short_name}}. Because system accounts are not associated with an email address and a user, they can be used for automation and integrations. 
+
+System accounts offer the following benefits over regular user accounts:
+
+* System accounts are not associated with an email address. This allows you to use the account as part of an automation or integration that isn't associated with any person’s identity.
+* When you use a user account as part of an automation or integration and that user leaves the company, automation and integrations break. If you use a system account instead, the automation and integrations wouldn't break.
+* System accounts don't have sign-in credentials and therefore can't access the {{site.konnect_short_name}} UI. These accounts are intended to be used with APIs and CLIs.
+
+The system account can use a {{site.konnect_short_name}} personal access token (PAT) the same way a [regular {{site.konnect_short_name}} user](/konnect/org-management/users/) can. In addition, the system account can be assigned roles directly or inherit the roles of a [team](/konnect/org-management/teams-and-roles/). As such, a PAT created by a system account inherits the roles assigned to the system account.
+
+## Manage a system account via the user interface
+
+You can create a system account using the {{site.konnect_short_name}} user interface by navigating to **Organization > System Accounts**. You can also assign system accounts to teams and manage system account tokens from this page.
+
```

Comment:
```suggestion
```
Remove the UI mention until UI becomes available.

Storing the original content here for when UI becomes available.
```
## Manage a system account via the user interface

You can create a system account using the {{site.konnect_short_name}} user interface by navigating to **Organization > System Accounts**. You can also assign system accounts to teams and manage system account tokens from this page.
```